src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int FIGURE_COUNT = 100;
```

Comment:
maybe we can find better name for `FIGURE_COUNT` constant,
because it isn't obvious what the figures we are talking about and why it is their count